yarn.lock

```diff
@@ -4781,6 +5110,239 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@smithy/abort-controller@^2.0.9":
```

Comment:
Looks like the `yarn.lock` has grown a lock, seems to be caused by these `@smithy/*` dependencies.  Could you find out where they are coming from?  It's a bit odd to get so many packages added by what should be minor updates in the `package.json`?

**Edit**: Looks like it comes from one of the aws dependencies, please have a look into what it's for and if maybe we can avoid that package to not grow our dependencies unnecessarily.  (Of course it's fine if necessary, but let's find out what's going on.)

Reply:
I'm trying to make it look better. But so far, this looks like ht reason: https://www.npmjs.com/package/@aws-sdk/smithy-client

As you can see, the package has moved. So what I'll try is to upgrade all the packages that were using the old one. In such case, I hope the size should be similar

Reply:
I think now it's much better. Same lines added, but before we were removing around 600, and now around 1200. The old `aws-sdk/smithy-client` is completely removed

Reply:
Interesting, so it's some code generator that the AWS people wrote.

Reply:
Yeah, and as the docs about that package claims, it shouldn't be used at all, at least directly. So probably something internal that just needed to be used in many places by them.

> **Edit**: What did you use to make yarn reduce the amount of dependencies?

That's something I still not sure what is the best way to go with. Basically, I remove the blocks from the yarn.lock and then execute `yarn install`, which picks the newest possible version. I remember seeing some other commands, but they were adding stuff to the package.json, so I discarded them